requirements.txt

```diff
@@ -8,7 +8,7 @@ airtable-python-wrapper==0.13.0
     # via wagtail-airtable
 anyascii==0.1.7
     # via wagtail
-asgiref==3.3.1
+asgiref==3.3.2
```

Comment:
Minor version bump in prep for Django 3.2. Seems to do no harm so far so I opted to keep this